### PR TITLE
Readme release

### PR DESCRIPTION
**Pull request by autra**
@iTowns/reviewers here is a small PR for the release.

It fixes globe.html and slightly rephrase the start of the README, and include a link to the doc and to some examples. 

This would need to be rework later (we probably want a http://www.itowns-project.org/itowns2/index.html listing the examples instead).

The goal is to have something ready before tomorrow :-) So I didn't make any extensive changes

### CODE REVIEW

**Comment by gchoqueux** [comment]
http://www.itowns-project.org/itowns2/index.html, example is broken?

File: examples/3dtiles.html [hunk 1]
```diff
@@ -78,7 +78,7 @@
                 itowns.$3dTilesSubdivisionControl
             );
             $3dTilesLayerDiscreteLOD.name = 'DiscreteLOD';
-            $3dTilesLayerDiscreteLOD.url = 'http://localhost:8003/tilesets/TilesetWithDiscreteLOD/tileset.json';
+            $3dTilesLayerDiscreteLOD.url = 'https://raw.githubusercontent.com/AnalyticalGraphicsInc/3d-tiles-samples/master/tilesets/TilesetWithDiscreteLOD/tileset.json';
```
Comment (peppsac) [inline examples/3dtiles.html:75]: With this change, you can remove the `readme` text at the beginning.

**Review by peppsac — APPROVED** [review]
Looks good, thanks

**Comment by autra** [comment]
> http://www.itowns-project.org/itowns2/index.html, example is broken? 

It is now at http://www.itowns-project.org/itowns2/examples/index.html (or globe.html, more or less the same). itowns2/index.html should be a landing page listing our examples, but I didn't have time to do that yet. Anyway it could be done after the release.

File: examples/3dtiles.html (hunk 1 above)
Comment (autra) [reply examples/3dtiles.html:75]: yep

File: README.md [hunk 2]
```diff
@@ -7,12 +7,20 @@
 
 ## What is it?
 
-iTowns is a web framework written in Javascript/WebGL for visualisation of 3D geographic data allowing precise measurements in 3D.
-Its first purpose was the visualisation of street view images and terrestrial lidar point cloud. It has then been extended to handle more data types.
+iTowns is a [Three.js](https://threejs.org/)-based framework written in Javascript/WebGL for visualizing 3D geospatial data allowing precise measurements.
```
Comment (Jeremy-Gaillard) [inline README.md]: The "allowing precise measurements" part seems a bit odd to men, since we don't have measuring tools yet.

Comment (Mkonini) [reply README.md]: http://www.itowns-project.org/itowns2/examples/globe.html 
doesn't run!

Comment (autra) [reply README.md]: @Mkonini yeah I know, as I'm fixing it is in this PR: see commit `Fix: examples/globe.html`

When this PR will be merged, it'll work again (you see, I care about the Globe ;-) ).
Ok another point still : why do you introduce cesium links on 3D Tiles exemple?

Comment (autra) [reply README.md]: @Jeremy-Gaillard I could probably delete this part yes.

**Comment by autra** [comment]
> http://www.itowns-project.org/itowns2/index.html, example is broken?

Btw it is independently of this release. I'll make a landing page tomorrow if I have time.

This PR fixes the globe @Mkonini .

@Jeremy-Gaillard I removed the part about the measurement.

**Comment by autra** [comment]
@gchoqueux @peppsac any objections against merging this?

**Comment by Mkonini** [comment]
@adevaux and @vcoindet have already modified the README brach, see  https://github.com/iTowns/itowns2/tree/README 
Did you took them into account?

**Comment by autra** [comment]
Yes I've even commented there. From above:

> The goal is to have something ready before tomorrow :-) So I didn't make any extensive changes

I don't think my work will disturb them in any way. Git, github and pull requests are made to work simultaneously on the same files :-) (They'll juste replace what they want).

The thing is: as their rewrite is apparently pretty big, as their PR is not even open and as the release was planned for today (even if it'll be probably tomorrow), it won't be ready on time. We need at least correct informations in this README, as it will also end up in the npm home page for this package. That's why I made this PR: it's just a very small and modest correction for the release.

@nosy-b (which *is* Alexandre, ADevaux is someone else) and @vcoindet, please don't hesitate to comment here if anything disturbs you.

@ADevaux you've been notified by error, sorry for the noise, you can go back to whatever you were doing right now ;-)

**Comment by Mkonini** [comment]
I see, but it would be damage to release with some modest corrections only, while a big rework much more representative is already available on README branch.

**Pull request merged**